libstuff/SLog.cpp

```diff
@@ -39,6 +39,14 @@ void SLogStackTrace(int level) {
     }
 }
 
+// If the param name is not in this whitelist, we will log <REDACTED> in addLogParams.
+static const set<string> PARAMS_WHITELIST = {
```

Comment:
NAB, but we could "normalize" this by making them all lowercase or something and check lowercase, although this really shouldn't happen with our style guide for variable naming.

Reply:
I don't think its a blocker since I don't think it should happen in practice

Reply:
I consider this a feature 😁